trade-finance-manager-ui/server/controllers/case/parties/helpers.js

```diff
@@ -44,10 +44,34 @@ const isEmptyString = (str) => {
   return false;
 };
 
+/**
+ * Extracts party name from the URL
+ * @param {String} url Request URL
+ * @returns {String} Party name
+ */
+const partyType = (url) => {
```

Comment:
I'm not sure why we need this helper function?

Reply:
right. just seen below. we leave it as it is for now, but I'm hoping we can simplify this. so maybe one to tackle later